src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -1189,12 +1190,16 @@ public boolean getAssumeStaticInheritanceRequired() {
    */
   transient ErrorHandler errorHandler;
 
+  public InstrumentOption instrumentForCoverageOption;
```

Comment:
Add comment explaining what it does.

Reply:
Also please make this private or package-private

public members are harder to refactor